app/src/main/java/org/permanent/permanent/ui/composeComponents/StorageCard.kt

```diff
@@ -29,28 +32,37 @@ fun StorageCard(
     spaceUsedBytes: Long,
     spaceTotalBytes: Long,
     spaceUsedPercentage: Int,
+    backgroundWhite: Boolean = false
```

Comment:
instead of using a variable backgroundWhite that doesn't tell the reader too much, maybe use an enum class like: StorageProgressType and pass that along. It would be easier to read.